x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorTestUtils.java

```diff
@@ -370,6 +370,10 @@ public static ConnectorStatus getRandomInvalidConnectorNextStatus(ConnectorStatu
         return randomFrom(invalidStatuses);
     }
 
+    public static String nullString() {
+        return null;
+    }
```

Comment:
This is needed as I cannot pass `null` to the request constructors, if I've two expecting one argument each (`StreamInput` and `String`) as it would be ambiguous, which one to take.

Reply:
lol Java, should we make it a static constant instead of function?